libdisk/format/amiga/pinball_fantasies.c

```diff
@@ -0,0 +1,120 @@
+/*
+ * disk/pinball_fantasies.c
+ * 
+ * Custom format for Pinball Fantasies from 21st Century
+ * 
+ * Written in 2022 by Keith Krellwitz
+ * 
+ * RAW TRACK LAYOUT:
```

Comment:
This raw track layout comment doesn't match the code. Stale or copied and not updated?

Reply:
Updated the track layout comments.  Forgot to update them